packages/hawtio/src/plugins/camel/routes-service.test.ts

```diff
@@ -70,7 +75,7 @@ describe('routes-service', () => {
   test('processRouteXml wrong mbean', async () => {
```

Comment:
So now it's a test for `getRoutesXml`, so please change the description as well.